pkg/cli/admin/ocpcertificates/regeneratemco/rotatecerts.go

```diff
@@ -125,6 +131,37 @@ func (o *RegenerateMCOOptions) Run(ctx context.Context) error {
 	return nil
 }
 
+func (o *RegenerateMCOOptions) ensureMCSSecretType(c *kubernetes.Clientset, ctx context.Context) error {
+	// Retrieve the machine-config-server-tls secret
+	mcsTLSSecret, err := c.CoreV1().Secrets(mcoNamespace).Get(ctx, mcsTlsSecretName, metav1.GetOptions{})
+	if err != nil {
```

Comment:
I assume that notfound error for this secret is not expected?

Reply:
It is not expected(nodes cannot join the cluster if this secret doesn't exist in the MCO namespace), but good callout - I think it might be good if the command accounted for this case